app/views/showApplication.scala.html

```diff
@@ -154,18 +154,18 @@ <h4 class="mdl-dialog__title">Est-ce que la réponse vous semble utile pour l'us
             <button type="button" disabled="disabled" class="mdl-button mdl-button--raised mdl-button--colored" id="close-dialog-terminate" onclick="confirmTerminate()">Clore</button>
         </div>
     </dialog>
-
-
-
+        <div class="mdl-cell mdl-cell--12-col mdl-cell--12-col-phone">
```

Comment:
ça serait mieux d'utiliser le positionnement du framework et de supprimer le float: right"

```suggestion
        <div class="mdl-cell  mdl-cell--2-col mdl-cell--10-offset-desktop mdl-cell--12-col-phone">
```

Reply:
J'ai utilisé 3 colonnes pour que le bouton soit entièrement affiché.